src/redux/missions/missionsSlice.js

```diff
@@ -0,0 +1,34 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'https://api.spacexdata.com/v3/missions';
+
+const initialState = {
+  missions: [],
+};
+
+export const fetchMissions = createAsyncThunk('missions/fetchMissions', async () => {
+  const response = await axios.get(url);
+  return response.data;
```

Comment:
- [ ] Kindly filter the incoming data before returning it for the reasons discussed earlier since we don't need the entire information coming from the API. We just need a small piece of information for the application to work. ✍️

**Project requirements**
![Screenshot from 2023-03-09 13-03-57](https://user-images.githubusercontent.com/17243325/223992572-d5fddf57-7fab-43f8-bd6d-c708c36a9bd5.png)